.eslintrc.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
     '@typescript-eslint/no-use-before-define': 'error',
     '@typescript-eslint/prefer-nullish-coalescing': 'error',
     '@typescript-eslint/prefer-ts-expect-error': 'error',
-    '@typescript-eslint/restrict-template-expressions': 'error',
+    '@typescript-eslint/restrict-template-expressions': ['error', { allowAny: true }],
```

Comment:
I'd like to keep this strict, please only ignore this in the corresponding line if necessary.